### DISCUSSION TITLE
React-window infinite-scroll

### DISCUSSION THREAD

**Discussion by abdullaev388** [rsuite/rsuite#2582] [General]
Hi i would like to use your Table with infinite-scroll from react-window
The problem is i have no idea how to do it.

here is it:
https://github.com/bvaughn/react-window-infinite-loader

**Comment by simonguo** [comment]
The issue will be converted to discussion.

**Reply by abdullaev388** [reply]
Here is the table with 100 rows: https://codesandbox.io/s/hopeful-swanson-wseq7z?file=/src/index.js
Please, can you integrate into this example react-window infinite scroll because i have no idea how to do it.

**Reply by simonguo** [reply]
rsuite table cannot be used with react-window. It is virtualized. You can listen to Table scroll events and load new data when the scroll bar is near the bottom.

Reference example:  https://rsuite.github.io/rsuite-table/#23
